sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -0,0 +1,116 @@
+﻿# Level-Zero backend specification
+
+## 1. Introduction
+
+This extension introduces Level-Zero backend for SYCL.
+It is built on top of Level-Zero runtime enabled with [Level-Zero API](https://spec.oneapi.com/level-zero/latest/index.html).
+The Level-Zero backend is aimed to provide the best possible performance of SYCL application on a variety of targets supported.
+The currently suppported targets are all Intel GPUs starting with Gen9.
+
+NOTE: This specification is a draft. While describing the currently implemented behaviors it is known to be not complete nor exhaustive.
+      We shall continue to add more information, e.g. explain general mapping of SYCL programming model to Level-Zero API.
+      It will also be gradually changing to a SYCL-2020 conforming implementation.
+
+## 2. Prerequisites
+
+The Level-Zero loader and drivers need to be installed on the system for SYCL runtime to recognize and enable the Level-Zero backend.
+For further details see [https://software.intel.com/content/www/us/en/develop/articles/intel-oneapi-dpcpp-system-requirements.html](https://software.intel.com/content/www/us/en/develop/articles/intel-oneapi-dpcpp-system-requirements.html).

```

Comment:
I'm not certain, but I think you can avoid duplicating the URL in cases like this by using this syntax:

```suggestion
For further details see <https://software.intel.com/content/www/us/en/develop/articles/intel-oneapi-dpcpp-system-requirements.html>.
```

Reply:
Fixed.